### PR TITLE
Initial project documentation: `README.md` and `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by xeho91**
**This Pull Request purpose is to start building initial documentation files for the project.**

---

## README.md

- [x] Short introduction
- [x] Define "Our Philosophy"
- [x] Mention our Wiki in the project
- [x] Invite to contributing to the project by mentioning [CONTRIBUTING.md] file
- [x] Add "Contact us" section

## CONTRIBUTING.md

- [x] Define "Prerequisities"
- [x] Mention "Coding style guidelines"
- [x] Define Git setup
- [x] Mention list of available scripts

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+We want to treat novices in the same way we wish we were treated when we began
+our adventure in writing code.
+
+---
+
+## 👣 Developing steps
+
+Before you decide to contribute, **please review our developing steps
+quickly**.\
+They're supposed to help all of us, to make collaboration and coding an
```
Comment (lax0248917) [inline CONTRIBUTING.md:37]: **Edit** ...all of us, to make collaboration and coding **"a"** joyous adventure.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+joyous adventure. So we did our best to simplify the processes as much as
+possible.
+
+### [Coding style guidelines]
+
+We focus on **code readability**. Take a look at our 📔 [coding style guidelines]
+on our Wiki page for some hints. This will also save our time during the Code
```
Comment (lax0248917) [inline CONTRIBUTING.md:44]: **Edit** ...This will also save **"your"** time during...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+### Git setup
+
+We have organized Git set up to make workflow enjoyable, and most importantly,
+manageable.
+
+#### [Git flow]
+
+Take a look at the 🫒 [Git flow] page on our Wiki in this project' repository.
+It thoroughly explains it and provides instructions on creating a new branch.
```
Comment (lax0248917) [inline CONTRIBUTING.md:57]: **Rephrase** ...thoroughly explains **"Git Flow"** and provides...

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+[coding style guidelines]: https://github.com/terminal-nerds/terminal-nerd/wiki/coding-style-guidelines
+
+### Git setup
+
+We have organized Git set up to make workflow enjoyable, and most importantly,
+manageable.
+
+#### [Git flow]
+
+Take a look at the 🫒 [Git flow] page on our Wiki in this project' repository.
```
Comment (lax0248917) [inline CONTRIBUTING.md:56]: **Edit** ...in this **"project's"** repository.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+It thoroughly explains it and provides instructions on creating a new branch.
+
+[git flow]: https://github.com/terminal-nerds/terminal-nerd/wiki/🫒-Git-flow
+
+#### [Git commits convention]
+
+[![Conventional Commits shield]][conventional commits] [![Gitmoji shield]][gitmoji]
+
+Take a look at the 💌 [Git commits convention] page on our Wiki in this
+project' repository.
```
Comment (lax0248917) [inline CONTRIBUTING.md:66]: **Edit** ...in this **"project's"** repository.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+It thoroughly explains it and provides instructions on how to write the commit
```
Comment (lax0248917) [inline CONTRIBUTING.md:67]: **Rephrase** ...thoroughly explains **" what the Git Commits Convention is"** and provides...

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+[conventional commits shield]: https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg?style=for-the-badge
+[conventional commits]: https://conventionalcommits.org "Conventional commits convention"
+[gitmoji shield]: https://img.shields.io/badge/gitmoji-%20😜%20😍-FFDD67.svg?style=for-the-badge
+[gitmoji]: https://gitmoji.dev "Gitmoji convention"
+[git commits convention]: https://github.com/terminal-nerds/terminal-nerd/wiki/git-commits-convention
+
+### [Scripts]
+
+Take a look at the 🧰 [Scripts] page on our Wiki in this project' repository.
```
Comment (lax0248917) [inline CONTRIBUTING.md:78]: **Edit** ... in this **"project's"** repository.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+---
+
+Placeholder for project idea
+
+---
+
+## 📚 [Wiki]
+
+**We put a lot of effort on our project [Wiki].** If you find within our
```
Comment (lax0248917) [inline README.md:17]: **Rephrase** ... a lot of effort **"into"** our...

File: README.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+project an abbrevation or jargon you don't understand, please take a look at
```
Comment (lax0248917) [inline README.md:18]: **Rephrase, Spelling Error**... project an **"abbreviation"** or **"some"** jargon you...

File: README.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+Placeholder for project structure
+
+---
+
+## 💕 [Contributing to the project]
+
+**If you wish to contribute to this project, please take a look at the
+[CONTRIBUTING.md](./CONTRIBUTING.md) file**. You will find instructions there
+how to navigate through our development processes.
```
Comment (lax0248917) [inline README.md:39]: **Rephrase**... instructions there **"on"** how to navigate...

File: README.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+[![discord shield]][discord server]
+
+**The best way to contact us is using our [Discord server]**. Anyone is welcome to
+hop in for a chat, if you don't feel comfortable using [Discussions]. 🙂
+
+[discord server]: https://discord.gg/decp3g7BEN
+[discord shield]: https://img.shields.io/discord/862890839537877012?label=Discord&logo=discord&style=for-the-badge
+[discussions]: https://github.com/terminal-nerds/terminal-nerd/discussions
+
+**We are also open to feedback**. If you have one or any suggestions, feel free
```
Comment (lax0248917) [inline README.md:66]: **Rephrase** ... If you have **"any suggestions"**, feel free...

File: README.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+to [start a discussion] in our project' repository or use "#💡-ideas" channel
```
Comment (lax0248917) [inline README.md:67]: **Edit** ...in our **"project's"** repository...

File: README.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+on our [Discord server] and share your thoughts & ideas!
+
+We would love to hear suggestions such as:
+
+-   how to simplify the processes of developing steps,
```
Comment (lax0248917) [inline README.md:72]: **???** Clarity on the meaning here.
Are we looking for help in simplifying the "Software" developing process? Or the process of developing software engineering steps? Sorry I'm a little confused lol

**Review by lax0248917 — CHANGES_REQUESTED** [review]
This is a great start! How you write markdown so effective and efficiently amazes me. I have proof-read and added a preliminary round of edits and rephrasing that I think will help. Look forward to your feedback as well.

File: README.md (hunk 13 above)
Comment (xeho91) [reply README.md:72]: I meant our defined Developing Steps in CONTRIBUTING.md, to make the steps simpler. Make sure is not too overcomplicated.

File: CONTRIBUTING.md (hunk 1 above)
Comment (xeho91) [reply CONTRIBUTING.md:37]: Done, woops!

File: CONTRIBUTING.md (hunk 2 above)
Comment (xeho91) [reply CONTRIBUTING.md:56]: Done.

File: CONTRIBUTING.md (hunk 5 above)
Comment (xeho91) [reply CONTRIBUTING.md:78]: Done.

File: README.md (hunk 9 above)
Comment (xeho91) [reply README.md:18]: Done.

File: README.md (hunk 10 above)
Comment (xeho91) [reply README.md:39]: Done.

File: README.md (hunk 12 above)
Comment (xeho91) [reply README.md:67]: Done.

File: README.md (hunk 11 above)
Comment (xeho91) [reply README.md:66]: Done.

File: CONTRIBUTING.md (hunk 3 above)
Comment (xeho91) [reply CONTRIBUTING.md:69]: Done.

File: CONTRIBUTING.md (hunk 7 above)
Comment (xeho91) [reply CONTRIBUTING.md:90]: Done.

File: CONTRIBUTING.md (hunk 6 above)
Comment (xeho91) [reply CONTRIBUTING.md:79]: Done.

File: README.md (hunk 8 above)
Comment (xeho91) [reply README.md:17]: Done.

File: README.md [hunk 14]
```diff
@@ -8,15 +8,24 @@ terminal nerds_.
...
 Placeholder for project idea
 
 ---
 
 ## 📚 [Wiki]
 
-**We put a lot of effort on our project [Wiki].** If you find within our
-project an abbrevation or jargon you don't understand, please take a look at
-our [Dictionary page] on the Wiki. There's a chance you may find a short
+**We put a lot of effort into on our project [Wiki].** If you find within our
```
Comment (lax0248917) [inline README.md:26]: This line should read as follows... "We put a lot of effort into our project Wiki...
Need to remove the word **"on"**

**Review by lax0248917 — CHANGES_REQUESTED** [review]
One minor change needs made, but the rest is spot on and looks ready to go!

File: README.md (hunk 13 above)
Comment (lax0248917) [reply README.md:81]: Perfect was just making sure. You can resolve this conversation then and we can leave it the way it is written!!  👍

**Pull request merged**